joomla/joomla-cms
# Keep user.css last in order

Pull Request for Issue #38345 .

### Summary of Changes
Currently user.css loaded after template.css. however the modules css goes after.
This patch make user.css to be as last css in loaded stylesheets, as long as it not violate any dependency.



### Testing Instructions
Apply patch. Create user.css.
Inspect the source page for list of loaded stylesheets. Check that user.css is last in the list.
Also follow #38345

### Actual result BEFORE applying this Pull Request
Not always last, when there a module css.


### Expected result AFTER applying this Pull Request
Always last


### Documentation Changes Required
No, I guess.
